src/dosext/mfs/mfs.c

```diff
@@ -2451,18 +2454,28 @@ GetRedirection(struct vm86_regs *state, u_short index)
 
 	/* set the high bit of the return CL so that */
 	/* NetWare shell doesn't get confused */
-	returnCX = drives[dd].read_only | 0x80;
+	returnCX = 0x80;
+
+        if (drives[dd].read_only)
+          returnCX |= 1;
+
+        /* see if DOS believes drive is redirected */
+        if (!GetCDSInDOS(dd, &tcds)) {
+          Debug0((dbg_fd, "GetRedirection can't get CDS\n"));
+        } else {
+          if ((cds_flags(tcds) & OUR_DRIVE_FLAGS) != OUR_DRIVE_FLAGS)
```

Comment:
Maybe only to check `REMOTE` here?
Btw, what does this `NOTNET` does?

Reply:
`REMOTE` only means network, `REMOTE | NOTNET` means cdrom. You have me wondering now if we should be setting NOTNET at all. I'll have a think.